### DISCUSSION TITLE
Is Integration with Django-CMS possible?

### DISCUSSION THREAD

**Discussion by pmi123** [DjangoCRM/django-crm#103] [Q&A]
Is it possible to integrate django-crm with django-cms? Suggestions on how it would be accomplished? Maybe with AppHook? [https://docs.django-cms.org/en/latest/introduction/05-apphooks.html](url)

**Comment by DjangoCRM** [comment]
Yes, it is possible.
But if it is for getting contact form data, then try [embedding the CRM form in the iframe](https://django-crm-admin.readthedocs.io/en/latest/setting_up_adding_requests/) of the CMS page.
It looks simpler.
